### PR TITLE
Interoperability: List of Countries can not be loaded for 1.5.x in new onboarding / interop (EXPOSUREAPP-2744)

### PR DESCRIPTION
**Pull request by BMItr**
## Relates to
https://jira.itc.sap.com/browse/EXPOSUREAPP-2744 (Overall)

## Description
This is an edge case which can mainly occur if during onboarding/delta onboarding or risk details config view the list of countries cannot be loaded due to missing internet connection.

## Quicktest
Start the App e.g. in Airplane Mode. Views will adjust according to screen specs (Flow Screenshot attached).
Inside the RiskDetails View is a new button card to open the Settings/Connectivity Settings if the country list could not be opened. Switch Back Airplane Mode to load the country list if possible.

## Note
This task based on Merts Work (new Interop / Countrylist PR #1202) and adjusted the ViewFlow in Onboarding/Delta/RiskDetails if neccessary.

## Screenflow
screenflow

### CODE REVIEW

File: Corona-Warn-App/src/main/res/layout/include_interoperability.xml [hunk 1]
```diff
@@ -151,37 +154,49 @@
...
+            <include
+                android:id="@+id/no_countries_default_infoview"
+                layout="@layout/include_interop_no_countries_infoview"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="@dimen/spacing_normal"
-                android:accessibilityHeading="true"
-                android:text="@{countryListTitle}"
-                android:visibility="@{FormatterHelper.formatVisibilityText(countryListTitle)}"
+                android:visibility="@{countryData.empty &amp;&amp; isOnboarding ? View.VISIBLE : View.GONE}"
```
Comment (MertSafter) [inline Corona-Warn-App/src/main/res/layout/include_interoperability.xml:163]: What do you think about a method in the FormatterHelper?

File: Corona-Warn-App/src/main/res/layout/include_interoperability.xml [hunk 2]
```diff
@@ -151,37 +154,49 @@
...
-            <de.rki.coronawarnapp.ui.view.CountryList
-                android:id="@+id/countryList"
+            <include
+                android:id="@+id/no_countries_riskdetails_infoview"
+                layout="@layout/include_interop_riskdetails_no_countries_infoview"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="@dimen/spacing_normal"
-                app:list="@{countryData}"
+                android:visibility="@{(countryData.empty &amp;&amp; !isOnboarding)  ? View.VISIBLE : View.GONE}"
```
Comment (MertSafter) [inline Corona-Warn-App/src/main/res/layout/include_interoperability.xml:174]: Same as above

**Review by mertsafter — CHANGES_REQUESTED** [review]
Everythink seems to work fine except one small thing in the Delta Onboarding screen for Interoperability. It shows, that there is no internet connection available but the "Open System settings" button isn't working
image

File: Corona-Warn-App/src/main/res/layout/include_interoperability.xml (hunk 1 above)
Comment (BMItter) [reply Corona-Warn-App/src/main/res/layout/include_interoperability.xml:163]: has the same effect, but I can put it into there to be align with the other calls in this layout

File: Corona-Warn-App/src/main/res/layout/include_interoperability.xml (hunk 2 above)
Comment (BMItter) [reply Corona-Warn-App/src/main/res/layout/include_interoperability.xml]: adjusted. now like everywhere.

**Review by mertsafter — DISMISSED** [review]

**Review by SamuraiKek — DISMISSED** [review]

**Review by janetback — CHANGES_REQUESTED** [review]
hi Mert, in  <string name="interoperability_onboarding_list_subtitle_failrequest_no_network">
please write Ihre with a capital 'I':
Möglicherweise wurde Ihre Internet-Verbindung unterbrochen. 
Thanks!

**Comment by BMItter** [comment]
fixed. l is now L (captial)

**Review by janetback — APPROVED** [review]
UA reviewed

**Review by mertsafter — APPROVED** [review]

**Review by SamuraiKek — APPROVED** [review]

**Pull request merged**